src/ChatLinksPage.css

```diff
@@ -0,0 +1,31 @@
+.chatLink__container{
+    display: flex;
+    flex-direction: column;
```

Comment:
no need

Reply:
Then how to align items in this chat container?

Reply:
parent style will do the necessary